modules/apps/portal-store/portal-store-gcs/src/main/java/com/liferay/portal/store/gcs/GCSStore.java

```diff
@@ -315,6 +315,27 @@ private String _getFileVersionKey(
 			fileName, StringPool.SLASH, versionLabel);
 	}
 
+	private GoogleCredentials _getGoogleCredentials() throws IOException {
+		String serviceAccountKey = _gcsStoreConfiguration.serviceAccountKey();
+
+		if (Validator.isBlank(serviceAccountKey)) {
+			if (_log.isInfoEnabled()) {
+				_log.info(
+					"No credentials set for GCS Store. Library will default " +
+						"to using Application Default Credentials or " +
+							"Workload Identity for auth");
+			}
+
+			return ServiceAccountCredentials.getApplicationDefault();
+		}
+
+		try (InputStream inputStream = new ByteArrayInputStream(
```

Comment:
No need to try-with-resource on `ByteArrayInputStream`. Its `close()` method is actually empty. Please inline this variable.